src/commcare_cloud/commcare_cloud.py

```diff
@@ -148,9 +148,67 @@ def make_parser(available_envs, formatter_class=RawTextHelpFormatter,
     return parser, subparsers, commands
 
 
+def make_changelog_parser():
+    # Parse the contents of the changelog dir
+    changelog_contents = []
+    files_to_ignore = ['0000-changelog.md', 'index.md']
+    changelog_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', '..', 'docs', 'changelog')
+    sorted_files = _sort_files(changelog_dir)
```

Comment:
could pass `files_to_ignore` to `_sort_files` to have them excluded from the start